input/groups/armor.json

```diff
@@ -3079,7 +3005,40 @@
         10862,
         10865
       ],
-      "Standalone: Headgear + Back Item Combo": [
+      "Standalone: Headgear + Combo - Aspect Eye": [
+        8208,
+        8194,
+        8195,
+        8228,
+        8213,
+        8230,
+        8217,
+        8205,
+        8222
+      ],
+      "Standalone: Headgear + Combo - Enchanted Dragon Crown": [
```

Comment:
I've softened groups a bit, so this might not be 100% accurate, but more or less, this is what qualifies for a group:

* Either a similar appearance, a similar naming pattern, or a crafting/achievement connection.
* If the above doesn't apply, a similar theme might suffice, as long as there's a common source of acquisition.
* At least three distinct collectibles (same armor with different weight doesn't count).
* Exception for categories with no subtype (back items, minis, novelties), when crafting is involved, two items suffice.
* Gem Store has an exception, since it's bloated as hell, so just two distinct collectibles is enough; however, they must be of different subtypes.

The gem store category has a special classification by subtype, so if there's a group with entries from a single subtype (headgear) they get moved below; the question is, where do we put sunglasses, under headgear, or further above, in the middle of everything else? Armor is kinda of an unique case, that's why it's such a mess.

**Grouped, work as combo:**

* Aspect Eye
* Enchanted Dragon Crown
* Pointed Ears

**Grouped, but don't work as combo:**

* Fuzzy Hat
* Glasses / Sunglasses
* Glowing Mask
* Mask of the Crown / Mask of the Jubilee / Mask of the Queen

The problem is there's just so much stuff there, if we move those bundles up, we should move the other bundles up as well, and headgear subtype information would be lost... still, if you rework combo into an icon, the scenario would be different.

I wouldn't think about this any further, it's a complete headache, wait for later, when the combo problem has been solved.

Reply:
Fair, I've applied your changes as is.